content/5_Plat/Bin_Jump.mdx

```diff
@@ -353,6 +353,76 @@ int main() {
 
 <IncompleteSection />
```

Comment:
remove this too

Reply:
i thought the incomplete section was for more implementation